decider-bcl2fastq/src/main/java/ca/on/oicr/pde/deciders/Bcl2fastqDecider.java

```diff
@@ -538,20 +574,20 @@ public void setConfig(Map<String, String> config) {
         //lane validation before scheduling workflow run
         Set<String> invalidLanes = new HashSet<>();
         for (String laneName : candidateLanesToAnalyze) {
-            List<String> errors = new ArrayList<>();
+            List<String> laneErrors = new ArrayList<>();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "laneErrors" which hides the field declared at line 106. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AHiddenFieldCheck)